example.md

```diff
@@ -0,0 +1,274 @@
+Plotting a Country's Daily Average CO-Values
+============================================
+
+In this example, we will be using the
+[Average API](https://demo.emissions-api.org/ui/#/default/emissionsapi.web.get_average)
+to request data for a given country,
+then we will plot this data using [Chart.js](https://chartjs.org).
+The result should look somewhat like this:
+
+
+<canvas id="average-example"></canvas>
+
+
+Average Data
+------------
+
+First, let's take a look at the data.
+For this, we [request the average carbon monoxide data of Germany for February 2019
+](https://demo.emissions-api.org/api/v1/average.json?country=DE&begin=2019-02-01&end=2019-03-01)
+using the parameters *begin*, *end* and *country*.
+
+```
+https://demo.emissions-api.org/api/v1/average.json
+    ?country=DE
+    &begin=2019-02-01
+    &end=2019-03-01
+```
+
+Note that *country* is just a convenient way of specifying a geographic bounding box for a specific country
+and we could also specify a polygon manually.
+Further note that we specify May 1st as end date
+since *end* represents the first date not being included in the resulting data.
+
+The response should look like this:
+
+```json
+[
+  {
+    "average": 0.0330089744766232,
+    "end": "2019-02-01T14:17:51.014000Z",
+    "start": "2019-02-01T12:34:43.435000Z"
+  },
+  {
+    "average": 0.032249495510353,
+    "end": "2019-02-02T13:58:41.698000Z",
+    "start": "2019-02-02T12:15:55.960000Z"
+  },
+  …
+]
+```
+
+What we get is a response in JSON format,
+listing the average value of carbon monoxide particles per square meter
+in the specified area.
+
+We also get the time of the first and last measurement made on each day in that area.
+But since we know that we are talking about an average per day, we can mostly ignore the specific time.
```

Comment:
I find it a valuable information that the measuring window is in that time frame. It shows that it is indeed *not* an average per day, but an average of a couple of hours. This poses a serious limitation of the data that should not be ignored.

```suggestion
We can infer from this information during what time of day the measurements where taken.
```

Reply:
True for the data and I was wondering if I should talk about this but when limiting the focus on just building this graph, this additional information is actually quite useless. There is a lot more you could say about the data that would be interesting or important, just not for the goal we have set in this guide.

'll partly include your remarks though.I